install-openqa-worker.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+set -e
+
+# Test script based on installation guide from Fedora:
+# https://fedoraproject.org/wiki/OpenQA_direct_installation_guide
+
+if [ -z "$1" ]
+  then
+    echo "No argument supplied, please enter OpenQA Server fqdn"
+fi
+
+
+release=$(uname -a)
+export release
+echo Installing OpenQA Worker on Fedora
+echo Running on: "$release"
+
+pkgs=(guestfs-tools libguestfs-xfs libvirt-daemon-config-network perl-REST-Client python3-libguestfs virt-install withlock)
+if ! rpm -q "${pkgs[@]}" &> /dev/null; then
+  sudo dnf install -y "${pkgs[@]}"
+else
+  echo "openqa and all requirements installed."
+fi
+
+# Open vnc port for 4 local worker clients
+sudo firewall-cmd --permanent --new-service=openqa-vnc
+sudo firewall-cmd --permanent --service=openqa-vnc --add-port=5991-5999/tcp
+sudo firewall-cmd --permanent --add-service openqa-vnc
+sudo firewall-cmd --permanent --new-service=openqa-socket
+sudo firewall-cmd --permanent --service=openqa-socket --add-port=20000-20089/tcp
+sudo firewall-cmd --permanent --add-service openqa-socket
+sudo firewall-cmd --reload
+
+if sudo grep -q foo /etc/openqa/client.conf; then
```

Comment:
This is basically the auth credential with which the worker connects itself to the main service 🙂

Reply:
I should be able to run this with little impact in the Fedora 36 VM I have waiting for openQA install. 

@lumarel , I presume I can run both the server and worker install script on the same VM.

A cursory glance suggests I should run the split scripts instead of the monolithic one followed by the post-rocky script but you didn't update the README.md which might be nice to do in this (or a subsequent) PR.